test/cpp_extensions/python_agnostic_extension/python_agnostic/csrc/tensor_core_tiled_layout.cu

```diff
@@ -0,0 +1,316 @@
+#if !defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 800  // at least Ampere
```

Comment:
+1 from the comment above that we can shrink this.
We just need something minimal that depends on cuda and aten right?

Reply:
yeh